### ISSUE TITLE
Writing only the package name when using the replace option

### ISSUE THREAD

**Issue by kentSarmiento** [TobyAndToby/generate-license-file#343]
https://generate-license-file.js.org/docs/cli/config-file#replace

For the replace option, is it possible to write only the name of the package, without including the version?

Current:
```
module.exports = {
  replace: {
    "lodash@1.2.3": "./licenses/lodash.txt",
    "react@1.2.3": "./node_modules/react/LICENSE",
  },
};
```

Suggestion:
```
module.exports = {
  replace: {
    "lodash": "./licenses/lodash.txt",
    "react": "./node_modules/react/LICENSE",
  },
};
```

**Comment by tobysmith568** [comment]
Hey @kentSarmiento ,

Sorry for the lack of reply here - I wanted to let you know that we have seen this issue and have been thinking about it for the last week or so.

While this could be an achievable feature request, we're currently unsure if it's something that we want to support (at least, as described). We say this because if a person were to replace all versions of a packages license then they'd also be silently replacing any future updates to that packages license without knowing it.

While uncommon, it's very possible for a package to update its license between major, minor, or patch releases and we suggest that users should always know what it is that they're replacing.

This being said, we do recognise the value in not having to continuously keep your generate-license-file config up to date with your package.json - especially if you were to have automated processes updating packages, like dependabot / renovate etc. - so this is for sure something we're still actively considering in some shape/form.

We'll be sure to reply here when we have an update. In the meantime, if you have any suggestions for how we can implement this more safely, then please let us know.

Thanks :)

**Comment by kentSarmiento** [comment]
Hi @tobysmith568 ,

Thank you for responding. :)

I think it would still be good to add that option. It should be the responsibility of users to check whether the license are changed or not (they should be able to use other tools to check licenses).

Best regards,

**Comment by tobysmith568** [comment]
Hey @kentSarmiento ,

We've spoken it over and agree that this a feature where it should be on the consumer to verify they they're working with their dependencies in a reasonably safe manner.

I've opened #359 and @tobybessant and I will get it merged and released over the next few days.

**Comment by kentSarmiento** [comment]
Awesome! Thank you @tobysmith568 and @tobybessant 😄

**Comment by tobybessant** [comment]
Hey @kentSarmiento ,

We've just released version 3.2.0 which includes this feature - please feel free to install it locally and let us know what you think!

Thanks for your suggestion/contribution to the project 🙂
